app/models/user.rb

```diff
@@ -88,14 +102,26 @@ def filter(query)
     end
   end
 
+
   has_many :user_identifiers
+
   has_many :identifier_schemes, through: :user_identifiers
+
   has_and_belongs_to_many :notifications, dependent: :destroy, join_table: 'notification_acknowledgements'
 
-  validates :email, email: true, allow_nil: true, uniqueness: {message: _("must be unique")}
 
-  ##
-  # Scopes
+  # ===============
+  # = Validations =
+  # ===============
+
+  validates :active, inclusion: { in: BOOLEAN_VALUES,
+                                  message: INCLUSION_MESSAGE }
+
+
```

Comment:
email should be unique. Devise may manage that for us though ... not sure

Reply:
Yep, we include the `validatable` module in devise.